packages/block-library/src/details/edit.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks, RichText } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+export default ( { attributes, setAttributes } ) => {
+	return (
+		<>
+			<RichText
+				tagName="div"
+				value={ attributes.content }
+				onChange={ ( content ) => {
+					setAttributes( { content } );
+				} }
+				placeholder={ __( 'Visible Content' ) }
+			/>
+			<hr />
```

Comment:
Semantically, I'm pretty sure this isn't how the [`hr` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr) is supposed to be used. I think it should be removed.